lib/suppressions.h

```diff
@@ -150,7 +160,7 @@ class CPPCHECKLIB SuppressionList {
         std::size_t hash{};
         bool thisAndNextLine{}; // Special case for backwards compatibility: { // cppcheck-suppress something
         bool matched{};
-        bool checked{}; // for inline suppressions, checked or not
+        bool checked{};
```

Comment:
could you add comments that explain "checked" and "matched".

Spontanously it sounds like "checked" just means that isSuppressed has been called at all? It does not say if the checkers has been executed or not?

And "matched" indicates if some warning was suppressed by this suppression?

Reply:
Will do.

Reply:
```cpp
        bool matched{}; /** This suppression was fully matched in an isSuppressed() call */
        bool checked{}; /** This suppression applied to code which was being analyzed but did not match the error in an isSuppressed() call */
```